### PR TITLE
ENT-9832: Long title text causes disposition of the gh button

### PR DESCRIPTION
**Pull request by mineralsfree**
image

Ticket: ENT-9832
Changelog: None

### CODE REVIEW

**Comment by mineralsfree** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/424/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/424/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-424/output/_site/

File: generator/_assets/styles/less/article.less [hunk 1]
```diff
@@ -110,10 +110,13 @@ article {
...
     .article_title {
       display: flex;
-      align-items: center;
+      align-items: flex-start;
+      word-wrap: break-word;
+      grid-gap: 12px;
 
       h1 {
         flex-grow: 1;
+        max-width: calc(100% - 110px);
```
Comment (aleksandrychev) [inline generator/_assets/styles/less/article.less:119]: in the phone-down this should be changed to 100%, otherwise on the mobile biew will look wrong

Comment (mineralsfree) [reply generator/_assets/styles/less/article.less:119]: updated commit

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/431/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/431/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-431/output/_site/

**Review by olehermanse — APPROVED** [review]

**Comment by mineralsfree** [comment]
Cherry-pick to 2.21: 
https://github.com/cfengine/documentation/pull/3133

**Pull request merged**